### ISSUE TITLE
Remove unused `alt-hero` field from 311-data.md project file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2892]
### Overview
As a developer, I want to make sure unused code is removed from the codebase in project cards so that
- time to load the website is as low as possible,
- it's easier to understand the code, and
- time to review the code is reduced.

### Details
In project md files, `image-hero` is a decorative background image and `alt-hero` is the alt text associated with `image-hero`. However, `alt-hero` text is not used anywhere in our codebase since the `image-hero` is used as an SCSS background image.

Moreover, if the `image-hero` were to be used in an img HTML tag in the future, the alt text for all the `image-hero` images would be `alt=""`, which could be set programmatically for all the alt properties associated with each `image-hero`. 

### Action Items
- [x] For the file `_projects/311-data.md`, remove line 8 for the alt-hero
```
alt-hero: 'Macbook Pro showing the 311 data app display. On the left of the screen shows input options and selections for dates, neighborhood council and various requests. On the right of the screen shows a map of LA county and its corresponding checked neighborhood council and requests.'
```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2722 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2722 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #2722 

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/311-data.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/d1f1ce97eb2cff948b0f2bee52158feba417a3ee/_projects/311-data.md?plain=1#L8

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

**Comment by trishajjohnson** [comment]
Availability for this week: 6hrs
My estimated ETA for completing this issue: end of day Thursday

**Comment by JessicaLucindaCheng** [comment]
@trishajjohnson Please give a progress report in a comment in this issue that includes Progress, Blockers, Availability, and ETA. See more detailed information in [section 2.5 Reporting progress on your issue in CONTRIBUTING.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#25-reporting-progress-on-your-issue).  Also, if you are taking a leave of absence or a vacation, please also let us know in your progress report.

**Comment by trishajjohnson** [comment]
Apologies, I got swallowed whole by some interviews/take-homes.

Progress: I need to push changes and create pull request to be reviewed/approved.
Blockers: None
Availability: 6hrs
ETA: pushed EOD 5/6, reviewed hopefully end of week
Pictures: None

**Comment by trishajjohnson** [comment]
@JessicaLucindaCheng I have 2 questions: when fetching upstream, do I need to be in the issue branch or gh-pages branch? Also, when preparing to push changes to my fork, do I add/commit/push from the issue branch or gh-pages?

**Comment by JessicaLucindaCheng** [comment]
@trishajjohnson
> when fetching upstream, do I need to be in the issue branch or gh-pages branch?

Answer: For fetching upstream, if you mean updating your repo, you would want to be in gh-pages

>  Also, when preparing to push changes to my fork, do I add/commit/push from the issue branch or gh-pages?

Answer: Your issue branch, which would be the branch you made for the issue you are working on.
